eng/azure-pipelines.yml

```
@@ -36,30 +36,39 @@ stages:
             x64:
               assetManifestOS: linux
               assetManifestPlatform: x64
-              imagename: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7-20220716123527-d0bc8ed
+              imagename: mcr.microsoft.com/dotnet-buildtools/prereqs:centos-7
               rootfs: 
+              clangtarget: 
               archflag: --arch x64
               LLVMTableGenArg: 
               ClangTableGenArg: 
               Devtoolset7Arg: /p:ForceDevtoolset7=true
+              ClangVersion: clang-9
+              ClangPlusVersion: clang++-9
```

please make the variable include the `/p:ClangVersion=` like this:
```suggestion
              ClangVersionArg: /p:ClangVersion=clang-9 /p:ClangPlusVersion=clang++9
```

The reason is in cases where `ClangVersion` is not defined (we don't have them here, but still) then Azure Pipelines wouldn't know how to replace the variable and pass the literal string `/p:ClangVersion=$(ClangVersion)` to the build which results in a warning.

This is also the reason why we did the same for `LLVMTableGenArg` etc